### ISSUE TITLE
Support for Icon in each dropdown option dynamically

### ISSUE THREAD

**Issue by JonathanDn** [JonathanDn/vue-dropdown#3]
Support is needed for rendering an icon inside each dropdown option by passing src's dynamically to config object

**Comment by tranlehaiquan** [comment]
You should let developer custom Option through slot. If you add icon name to attribute of Select, later developer will ask for left icon, right icon, custom style... You can read documentation of Element vue, it perfect one to : ). (Don't try grab all feature).
https://element.eleme.io/#/en-US/component/select

**Comment by JonathanDn** [comment]
Do you mean that i should allow for a HTMl slot to replace the existing content entirely and allow full customization from outside the component? 

In terms of motivation if I get what your saying correctly you mean:
- People have various requirements and needs from how the inner side of the option is fulfilled,
instead of trying to address each scenario --> enable a higher level of customization by allowing
to inject a slot element instead of the inner content of each option

Let me know if i understand what your suggestions is and if there is anything else you would like to add up?

**Comment by tranlehaiquan** [comment]
Yes, It what i try to say. You can read `https://element.eleme.io/#/en-US/component/select` at `Custom template` to see idea.
Sorry about my English : ), try to improve it.

**Comment by JonathanDn** [comment]
All is well my friend :), I just wanted to make sure that I understand your issue.

I think its a good idea to introduce this new functionality, I will aspire to introduce it to the plugin as soon as possible and thanks for the valuable feedback!
